src/guide/migration/events-api.md

```diff
@@ -58,6 +58,6 @@ We removed `$on`, `$off` and `$once` methods from the instance completely. `$emi
 
 ## Migration Strategy
 
-Existing event hubs can be replaced by using an external library implementing the event emitter interface, for example [mitt](https://github.com/developit/mitt).
+Existing event hubs can be replaced by using an external library implementing the event emitter interface, for example [tiny-emitter](https://github.com/scottcorgan/tiny-emitter).
```

Comment:
**suggestion**: would you mind if we leave both of them here? So people could make the choice and docs won't feel like we're promoting a certain library (we definitely made this mistake with `mitt`)?

```suggestion
Existing event hubs can be replaced by using an external library implementing the event emitter interface, for example [mitt](https://github.com/developit/mitt) or [tiny-emitter](https://github.com/scottcorgan/tiny-emitter).
```

Reply:
@NataliaTepluhina make sense and updated the PR.